Sources/Regex/Parse/CharacterPropertyClassification.swift

```diff
@@ -0,0 +1,392 @@
+import Util
+
+extension Source {
+  static private func normalizeForCharacterPropertyMatching(_ str: String) -> String {
+    // This follows the rules provided by UAX44-LM3, except the dropping
+    // of an "is" prefix, which UTS#18 RL1.2 states isn't a requirement.
+    return str.filter { !$0.isWhitespace && $0 != "_" && $0 != "-" }
+              .lowercased()
```

Comment:
FWIW, it would be nice to drop the `is` too, since `Unicode.Scalar.Properties` uses `is` and so does API on Character. Not sure if we'd have perfect parity with the other parts of the name, but I don't think it's harmful

Reply:
Updated to drop `is`, how does it look?